modules/ROOT/pages/multi-language-pages.adoc

```diff
@@ -41,15 +41,15 @@ To use a localization asset, replace the default `localization.json` file with a
   ...
 ----
 
-The code that specifies a language in `localization.json` must be the same as the code in the `.po` files used to xref:languages.adoc[add the language to the Portal].
+The code that specifies a language in `localization.json` must be the same as the code in the `.po` files used to xref:languages.adoc#_add_a_new_language[add a new language].
 
 By default, US english strings are used as the keys for translation. But you could use any code or language for the keys, and create an en-US section of translations to consider english as any other translated language. That option allows to freely change english strings with no duplication on every other language section.
```

Comment:
By default, US english strings are used as the keys for translation. But you could use any code or language for the keys, and create an en-US section of translations to consider english as any other translated language. That option allows to freely change english strings with no duplication on every other language section. => By default, translation keys are in en-US english texts. If you want to have custom texts, you can create a new translation section for en-US and handle it as a new translation. This option allows to freely change texts by avoiding duplicates on other language sections.

Reply:
Next time, you can use the "suggestion" feature to directly propose changes. (and thanks for the english phrasing... we are not great for that 😅  )
![Screenshot 2021-09-29 at 10 28 27](https://user-images.githubusercontent.com/2739234/135231915-8d0d69c8-30e2-49e3-aedc-54b1f15c622d.png)